watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1595_photothermal_membrane_candoP/amo_1595.py

```diff
@@ -0,0 +1,330 @@
+        expr=pyunits.convert(
+            m.fs.costing.bcp_recovery_mass * m.fs.costing.bcp_cost,
+            to_units=m.fs.costing.base_currency / m.fs.costing.base_period,
+        ),
+        doc="Dollar value of BCP generated per year",
+    )
+
+    m.fs.costing.LCOW_bcp = Expression(
+        expr=(
+            m.fs.costing.total_capital_cost * m.fs.costing.capital_recovery_factor
+            + m.fs.costing.total_operating_cost
+            - m.fs.costing.value_bcp_recovery
+        )
+        / (
+            pyunits.convert(
+                m.fs.photothermal_water.properties[0].flow_vol,
+                to_units=pyunits.m**3 / m.fs.costing.base_period,
+            )
+            * m.fs.costing.utilization_factor
+        ),
+        doc="Levelized Cost of Water when accounting for salable BCP",
+    )
+
+    m.fs.costing.LCOT = Expression(
+        expr=(
+            m.fs.costing.total_capital_cost * m.fs.costing.capital_recovery_factor
+            + m.fs.costing.total_operating_cost
+            - m.fs.costing.value_bcp_recovery
+        )
+        / (
+            pyunits.convert(
+                m.fs.feed.properties[0].flow_vol,
+                to_units=pyunits.m**3 / m.fs.costing.base_period,
+            )
+            * m.fs.costing.utilization_factor
+        ),
+        doc="Levelized Cost of Treatment (accounts for salable BCP)",
+    )
+
+
+def display_costing(m):
+    print("\n+++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
+    print("++++++++++++++++++++ DISPLAY COSTING ++++++++++++++++++++")
+    print("+++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
+
+    print("\n------------- Unit Capital Costs -------------")
+    for u in m.fs.costing._registered_unit_costing:
+        print(
+            f"{u.name} :   {value(pyunits.convert(u.capital_cost, to_units=pyunits.USD_2020)):.3f} $"
+        )
+
+    print("\n------------- Utility Costs -------------")
+    for f in m.fs.costing.flow_types:
+        print(
+            f"{f} :   {value(pyunits.convert(m.fs.costing.aggregate_flow_costs[f], to_units=pyunits.USD_2020 / pyunits.year)):.3f} $/year"
+        )
+
+    print("\n---------------------------------------")
+
+    total_capital_cost = value(
+        pyunits.convert(m.fs.costing.total_capital_cost, to_units=pyunits.MUSD_2020)
+    )
+    print(f"Total Capital Costs: {total_capital_cost:.3f} M$")
+
+    total_operating_cost = value(
+        pyunits.convert(
+            m.fs.costing.total_operating_cost, to_units=pyunits.MUSD_2020 / pyunits.year
+        )
+    )
+    print(f"Total Operating Costs: {total_operating_cost:.3f} M$/year")
+
+    electricity_intensity = value(
+        pyunits.convert(
+            m.fs.costing.electricity_intensity, to_units=pyunits.kWh / pyunits.m**3
+        )
+    )
+    print(f"Electricity Intensity: {electricity_intensity:.3f} kWh/m^3")
+
+    LCOW = value(
+        pyunits.convert(m.fs.costing.LCOW, to_units=pyunits.USD_2020 / pyunits.m**3)
```

Comment:
What is the flow basis for this LCOW metric?

Reply:
Treated water leaving the photothermal membrane unit. This is considered a byproduct, for this process.